lib/bindings/python/rust/engine.rs

```diff
@@ -246,7 +248,7 @@ where
                 };
 
                 if tx.send(response).await.is_err() {
-                    tracing::trace!(
+                    tracing::debug!(
                         request_id,
                         "error forwarding annotated response to channel; channel is closed"
                     );
```

Comment:
Note: A possible alternative into writing our own into_stream_with_locals_v1 method is to raise the exception into the Python generator object here, using run coroutine threadsafe. Will need to experiment if this alternative method works, but please comment if you prefer this alternative.